lametro/templates/search/search.html

```diff
@@ -1,6 +1,279 @@
 {% extends "search/search.html" %}
 
 {% load staticfiles %}
+{% load extras %}
+
+{% block content %}
```

Comment:
i had to override the entire content block in order to add the result type filter. i think this is ok, especially since we will probably be doing additional search customizations for metro moving forward. will leave comments where i actually made changes!